DD/Views/Partials/homePage/_navigation.cshtml

```diff
@@ -18,46 +17,40 @@
 <head>
     @await Html.PartialAsync("~/Views/Partials/_metaData.cshtml")
     <link rel="stylesheet" href="~/Styles/navigation.css" />
+
 </head>
 <body>
     <nav id="navigation">
         <div>
+            <a for="check" class="open-menu"><i class="fas fa-bars"></i></a>
             <ul>
                 @if (allVisiblePages != null && allVisiblePages.Any())
+                {
+                    var dropdownId = 0;
+                    foreach (var item in allVisiblePages)
                     {
-                        var dropdownId = 0;
-                @foreach (var item in allVisiblePages)
+                        var hasChildLinks = item.Name == "Emprendimiento" || item.Name == "Capacitacion" || item.Name == "Ferias y Eventos" || item.Name == "Sector Industrial" || item.Name == "Apoyos y Financiamientos";
```

Comment:
Avoid magic strings , and always compare strings using toLower inboth cases https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings

Reply:
I already changed the way to search the names of navigation bar in a different file.

Reply:
> I already changed the way to search the names of navigation bar in a different file.
Can confirm